src/overlays/actors/ovl_En_Reeba/z_en_reeba.c

```diff
@@ -49,8 +49,13 @@ const ActorInit En_Reeba_InitVars = {
 };
 
 static ColliderCylinderInit sCylinderInit = {
-    { COLTYPE_UNK5, 0x11, 0x09, 0x39, 0x10, COLSHAPE_CYLINDER },
-    { 0x00, { 0xFFCFFFFF, 0x08, 0x08 }, { 0xFFCFFFFF, 0x00, 0x00 }, 0x01, 0x05, 0x01 },
+    { COLTYPE_HIT5, AT_ON | AT_ENEMY, AC_ON | AC_PLAYER, OC_ON | OC_ALL, OT_TYPE1, COLSHAPE_CYLINDER },
+    { ELEMTYPE_UNK0,
```

Comment:
im kinda feeling like parens around the flags with | will make it easier to see which belong to which, but want to know what others think